docs/reference/pvc-finalizer-downgrade-issue.md

```diff
@@ -0,0 +1,32 @@
+* TOC
+{:toc}
+
+---
+title: Kubernetes Downgrade issue from 1.10 to 1.9 due to PV/PVC Protection
+---
+
+## PV/PVC Protection in Kubernetes 1.10
+
+When enabled, [PV/PVC Protection](docs/admin/admission-controllers.md#storage-object-in-use-protection-beta) prevents PV/PVC from being removed when the finalizers are removed. 
```

Comment:
The link should be `/docs/admin/admission-controllers.md#storage-object-in-use-protection-beta`, otherwise it will be broken. NB the first slash.

Reply:
thanks!